__tests__/app.test.js

```diff
@@ -411,5 +411,96 @@ describe("GET - 200: /api/reviews/:review_id", () => {
   });
 });
 
-
-  
+describe("GET - /api/reviews", () => {
+  test("GET - 200: responds with an array of review objects sorted by date DESC", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeSortedBy("created_at", { descending: true });
+        expect(body.length > 0).toBe(true);
+        body.forEach((review) => {
+          expect(review).toEqual({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+            comment_count: expect.any(String),
+          });
+        });
+      });
+  });
+  test("GET - 200: Responds with an array of reviews sorted by category", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=votes")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length > 0).toBe(true);
+        expect(body).toBeSorted("votes");
+        body.forEach((review) => {
+          expect(review).toEqual({
+            owner: expect.any(String),
+            title: expect.any(String),
+            review_id: expect.any(Number),
+            category: expect.any(String),
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            designer: expect.any(String),
+            comment_count: expect.any(String),
+          });
+        });
+      });
+  });
+  test("GET - 200: Responds with an array of reviews sorted by category", () => {
+    return request(app)
+      .get("/api/reviews?category=dexterity")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length > 0).toBe(true);
+        body.forEach((review) => {
+          expect(review).toEqual({
+            category: "dexterity",
+            comment_count: expect.any(String),
+            created_at: expect.any(String),
+            designer: expect.any(String),
+            owner: expect.any(String),
+            review_id: expect.any(Number),
+            review_img_url: expect.any(String),
+            title: expect.any(String),
+            votes: expect.any(Number),
+          });
+        });
+      });
+  });
+  describe("-error tests", () => {
+    test("GET - 404: responds with an error message, (Not found!), when passed an invalid endpoint", () => {
```

Comment:
If you already tested this previously, you don't need this for each endpoint. It's an error that covers all endpoints, not just this one specifically